source/crud/crud.rst

```diff
@@ -510,7 +510,7 @@ Read
      *
      * @see https://docs.mongodb.com/manual/reference/command/find/
      */
-    showRecordId: Optional<Document>;
+    showRecordId: Optional<Boolean>;
```

Comment:
I think this is minor enough to not require a last modified bump or changelog entry.

Reply:
Agreed.